app/src/docs/v1.api-spec.yaml

```diff
@@ -17,30 +17,348 @@ security:
   - BearerAuth: []
     OpenID: []
 paths:
-  /hello:
+  '/':
     get:
-      summary: Returns hello world result
-      operationId: getHello
+      summary: List all user accessible objects
+      description: >-
+        This endpoint returns a list of all user accessible objects.
+      operationId: GetAccessibleObjects
+      tags:
+        - Object
+      responses:
+        '200':
+          description: Returns the object information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ObjectDetails'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '422':
+          $ref: '#/components/responses/UnprocessableEntity'
```

Comment:
i dont think we need this response

Reply:
removing this response for all GET requests